app/public/js/controllers/activities.js

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright 2016 e-UCM (http://www.e-ucm.es/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * This project has received funding from the European Union’s Horizon
+ * 2020 research and innovation programme under grant agreement No 644187.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0 (link is external)
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+angular.module('activitiesApp', ['ngStorage', 'services'])
+    .controller('ActivitiesCtrl', ['$scope', '$location', '$http', '$window', '$localStorage', 'Games', 'Versions',
+        'Activities', 'Role', 'CONSTANTS', 'QueryParams',
+        function ($scope, $location, $http, $window, $localStorage, Games, Versions, Activities, Role, CONSTANTS, QueryParams) {
+            $scope.$storage = $localStorage;
+            $scope.activity = {};
+            $scope.class = {};
+            $scope.game = {};
```

Comment:
Removed in: https://github.com/e-ucm/rage-analytics-frontend/pull/90/commits/572bff82cfdb94cdb0df0d32cef2344ce44ca856